templates/zerver/emails/email.css

```diff
@@ -3,7 +3,8 @@
 because hex color codes are rendered by all email clients
 consistently.  Whereas HSL colors don't work with Hotmail. */
 
-img {
+.illustration img,
+td.wrapper img {
```

Comment:
I added a class for the one image `td.wrapper img` seems like it covers, so this can be just `img.bottom-illustration`.